learn/cloud/monitoring.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Monitoring metrics
+description: Meilisearch Cloud offers in-depth metrics to help monitor your application performance.
+sidebarDepth: 3
+---
+
+# Monitoring
+
+Enable Meilisearch Cloud monitoring to keep track of application performance and service status. 
+
+## Requirements
+
+You must have a [Meilisearch Cloud](https://meilisearch.com/cloud?utm_campaign=oss&utm_source=docs&utm_medium=monitoring) account to access monitoring metrics.
+
+## Activating monitoring
+
+### Enable monitoring in the project overview
+
+Log into your Meilisearch Cloud account and navigate to your project's overview. Find the "Analytics and monitoring" section and click on the "Enable analytics and monitoring" button:
+
+![The analytics section of the project overview. It shows one button, "Enable analytics and monitoring", and a short explanation of both features.](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/cloud-analytics/01-analytics-enable.png)
+
+Meilisearch Cloud will begin processing your request. The "Analytics and monitoring" section will update when the feature is enabled.
```

Comment:
Is it not only "Monitoring" section now?

Also, i think instead of "will update" it would be beneficial to explain how it will update, eg. "you will be able to see graphs" or "the tab will be accessible". if that makes sense

Reply:
As of 2024-02-05, the staging interface still reads "Analytics and monitoring". This is the content section beneath "API keys", not the tab at the top of the page.

Reply:
ty for clarification!